tests/fixtures/saas/iterable_fixtures.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+
+from tests.ops.integration_tests.saas.connector_runner import (
+    ConnectorRunner,
+    generate_random_email,
+)
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("iterable")
+
+
+@pytest.fixture(scope="session")
+def iterable_secrets(saas_config) -> Dict[str, Any]:
+    return {
+        "domain": pydash.get(saas_config, "iterable.domain")
+        or secrets["domain"],
+        "api_key": pydash.get(saas_config, "iterable.api_key") or  secrets["api_key"],
+        "email": pydash.get(saas_config, "iterable.email") or  secrets["email"],
```

Comment:
Can you remove this `email` secret if we're not using anywhere? I might have missed where we use it, let me know!